### PR TITLE
[Refactor]: change config content

### PR DESCRIPTION
**Pull request by yuewangg**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/faster_rcnn/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ We trained with R-50-FPN pytorch style backbone for 1x schedule.
...
-| R-50-FPN |    GIoULoss    |          |                |  37.6  |   [config](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py)   |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
+| R-50-FPN |    GIoULoss    |          |                |  37.6  |  [config](https://github.com/open-mmlab/mmdetection/blob/3.x/configs/faster_rcnn/faster-rcnn_r50_fpn_giou_1x_coco.py)  |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
```
Comment (RangiLyu) [inline configs/faster_rcnn/README.md:41]:
```suggestion
| R-50-FPN |    GIoULoss    |          |                |  37.6  |  [config](./faster-rcnn_r50_fpn_giou_1x_coco.py)  |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
```

**Review by RangiLyu — APPROVED** [review]

**Comment by ZwwWayne** [comment]
Hi @yuewangg ,
Thanks for your kind contribution. Would you like to fix the lint issue so that we could merge this PR?
The details can be found by clicking the item of lint check as shown below
image

**Comment by yuewangg** [comment]
I have submitted it again, please check it.

File: configs/faster_rcnn/README.md (hunk 1 above)
Comment (RangiLyu) [inline configs/faster_rcnn/README.md:41]:
```suggestion
| R-50-FPN |    GIoULoss    |          |                |  37.6  |  [config](./faster-rcnn_r50_fpn_giou_1x_coco.py)  |            [model](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco-0eada910.pth) \| [log](https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_giou_1x_coco_20200505_161120.log.json)            |
```

Comment (yuewangg) [reply configs/faster_rcnn/README.md:41]: I submitted your suggestion last time, but could not pass the lint test

**Pull request merged**